metrics/lib/features/dashboard/presentation/widgets/titled_text.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:metrics/features/dashboard/presentation/widgets/expandable_text.dart';
+
+/// The widget stands for displaying the [value] text with some [title].
+class TitledText extends StatelessWidget {
```

Comment:
Maybe - ValueBox? Or TextMetric? I have a feeling that there should be a better name for it.